README.md

```diff
@@ -31,4 +31,4 @@ What you can make:
 
 ### If you want to contribute a new template you have to follow our [Template Guidelines](https://github.com/mudit023/templates/blob/main/TEMPLATE_GUIDELINES.md).
 
-### Want to contribute to the project? Read the [Contribution Guidelines](https://github.com/mudit023/templates/blob/main/CONTRIBUTING.md) and adhere to our [Code of Conduct](https://github.com/mudit023/templates/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
keep your fork repo up to date. There are some changes made in Project README file, don't undo those.

Reply:
You want to delete line 34 ??? Because I don't understand

Reply:
@mudit023

Reply:
change 
1. "contribution guidelines" to " Contribution Guidelines" 
2.  "code of conduct" to "Code of Conduct"
3. put a full stop sign (period) at the end of line 34

Reply:
> change
> 
> 1. "contribution guidelines" to " Contribution Guidelines"
> 2. "code of conduct" to "Code of Conduct"
> 3. put a full stop sign (period) at the end of line 34

I don't understand, Please Make it